README.md

```diff
@@ -1 +1,57 @@
-# vcmi-dependencies
\ No newline at end of file
+# VCMI Dependencies
+
+This repository contains prebuilt Conan dependencies for vcmi. These dependencies are primary used by Github CI and can be used to avoid building dependencies locally by developers.
+
+Current flow to update dependencies:
+- push new version of conanfile and/or CI settings to vcmi/vcmi repository, to branch 'update_prebuilts'
+- run CI in this repository to generate new packages
+- create new release from generated artifacts
+- open PR in vcmi/vcmi with changes to conanfile, CI, new URL to dependencies package and whatever changes need to be done to source code to use newer package (if any)
+
+# TODO List
+
+- Find a way to set up an Artifactory on our server or find somebody willing to host it for us (e.g. JFrog) and deprecate this repository
+
+- Switch to conan 2. Incomplete (and potentially outdated) PR can be found here: https://github.com/vcmi/vcmi/pull/1603
```

Comment:
note: in Conan v2 we can't just zip the `data` directory, instead `conan cache` command must be used